ddtrace/constants.py

```diff
@@ -50,6 +50,14 @@
 
 PID = "process_id"
 
+SPAN_KIND = "span.kind"
+
+# Types of span.kind
+SPAN_CLIENT = "client"
+SPAN_SERVER = "server"
```

Comment:
Do we expect ddtrace users to use these constants? If these constants will only used in integrations we should define these values in an internal module (ex: ddtrace/internal/constants.py).

Suggestion: We can create a SpanKind class that defines these constants. This better aligns with how we handle [spantypes](https://github.com/DataDog/dd-trace-py/blob/v1.7.2/ddtrace/ext/__init__.py):

ex:
```
Class SpanKind:
    SPAN_CLIENT = "client"
    SPAN_SERVER = "server"
    SPAN_PRODUCER = "producer"
    SPAN_CONSUMER = "consumer"
```

We can define SpanKind in a public or internal module depending on the expected usage.

Reply:
Though these constants are mainly for our internal use, it's not out of the question IMO that a user would want to utilize these constants when manually instrumenting their own code. I like the class idea!

Reply:
Done. Added the SpanKind class like SpanTypes